cairo/plan.sh

```diff
@@ -38,7 +51,7 @@ do_build() {
   CFLAGS="-Os ${CFLAGS}"
 
   ./configure --prefix="${pkg_prefix}" \
-              --disable-xlib
+              --enable-xlib
```

Comment:
Is this needed? I think configure will automatically detect the availability of Xlibs.

Reply:
I like things to be written explicitly, but that's just me ;)